index.html

```diff
@@ -262,10 +262,7 @@ <h1 class="hidden-xs">NewPipe</h1>
                                 <div class="container">
                                     <div class="details-flex">
                                         <div class="details-image">
-                                            <figure>
-                                                <img src="img/Privacy_written_in_tiles.jpg"/>
-                                                <figcaption>Photo credit: <a href="www.homewatersoftenerreviews.com">Home Water Softener Reviews</a>. License: <a href="https://creativecommons.org/licenses/by/2.0/">CC BY 2.0</a></figcaption>
-                                            </figure>
+                                            <img src="img/Privacy_written_in_tiles.jpg"/>
```

Comment:
Correct me if I'm wrong, but it looks like you removed the credit on GitHub. That doesn't make very much sense.